blamo/lib/LogInfo/loginfo.dart

```diff
@@ -157,9 +158,10 @@ class _LogInfoPageState extends State<LogInfoPage> {
                               textInputAction: TextInputAction.next,
                               focusNode: formNodes[1],
                               attribute: 'number',
-                              validators: [FormBuilderValidators.numeric()],
```

Comment:
I am not certain, but I think we should keep the numeric validator?

Reply:
Looking at the examples, the input for that field is not strictly numbers. Which is why I removed that.